share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -157,6 +159,19 @@ function(rocm_install_targets)
                         NAMELINK_ONLY
             )
         endif()
+        if(T_TYPE MATCHES ".*_LIBRARY" AND ROCM_SYMLINK_LIBS AND NOT ROCM_CREATED_${PROJECT_NAME}_SYMLINK
```

Comment:
Is `ROCM_CREATED_${PROJECT_NAME}_SYMLINK` variable to only insert the symlink once?

Reply:
Yes. It shouldn't have any side effects if it were to be inserted multiple times, but I wanted to guard against it anywya.